chart/values.yaml

```diff
@@ -55,7 +55,7 @@ rabbitmq:
     #   name:
     #   key:
   # port defines the port of the rabbitmq server
-  port: 5672
+  port:
```

Comment:
You removed the default value? Does this mean that the helmchart is not deployable without specifying a port, now?

Is this the intended behavior?

A lot of "out-of-the-box" configuration, is slowly being removed. Should it not be possible to just deploy the helmchart, without specifying too many environment variables, in order to try out the product?

Reply:
> You removed the default value? Does this mean that the helmchart is not deployable without specifying a port, now?

Correct.

> Is this the intended behavior?

It seems it has to be. Helm templating spits out an error if you try to replace a leaf node with a scalar with a branch node with more configuration.

> A lot of "out-of-the-box" configuration, is slowly being removed. Should it not be possible to just deploy the helmchart, without specifying too many environment variables, in order to try out the product?

It does not seem too bothersome to provide all the required values for connecting to a database you have to setup yourself anyway.